react/src/components/UserCredentialList.tsx

```diff
@@ -511,6 +541,28 @@ const UserCredentialList: React.FC = () => {
           });
         }}
       />
+      <KeypairSettingModal
+        keypairSettingModalFrgmt={keypairSettingModalFrgmt}
+        open={!!keypairSettingModalFrgmt || openUserKeypairSettingModal}
+        onRequestClose={(success) => {
+          setKeypairSettingModalFrgmt(null);
+          setOpenUserKeypairSettingModal(false);
+          if (success) {
+            startRefreshTransition(() => {
+              updateFetchKey();
+            });
+          }
+        }}
+      />
+      <Suspense fallback={null}>
+        <KeypairInfoModal
+          keypairInfoModalFrgmt={keypairInfoModalFrgmt}
+          open={!!keypairInfoModalFrgmt}
+          onRequestClose={() => {
+            setKeypairInfoModalFrgmt(null);
+          }}
+        />
+      </Suspense>
     </Flex>
```

Comment:
Please check how to handle the info modal opening transition in `UserInfoModal` and apply the same logic.
https://github.com/lablup/backend.ai-webui/blob/8ada9ff919ea2e9781c614b9f7b6085e063ecfc5/react/src/components/UserNodeList.tsx#L400-L407